package.json

```diff
@@ -4,6 +4,9 @@
   "description": "A resilience and transient-fault-handling library that allows developers to express policies such as Backoff, Retry, Circuit Breaker, Timeout, Bulkhead Isolation, and Fallback in a fluent and thread-safe manner. Inspired by .NET Polly.",
   "main": "dist/index.js",
   "module": "dist/esm/index.js",
+  "engines": {
+    "node": ">=10 <11 || >=12"
```

Comment:
Wouldn't just `>=10` work?

Reply:
Yes, but technically that allows `11`. Not that it matters a heck of a lot.